aio/src/app/layout/doc-viewer/doc-viewer.component.ts

```diff
@@ -189,14 +189,15 @@ export class DocViewerComponent implements OnDestroy {
       const seconds = Number(cssValue.replace(/s$/, ''));
       return 1000 * seconds;
     };
+
+    // Some properties are not assignable and thus cannot be animated.
+    type StringValueCSSStyleDeclaration
+      = { [K in keyof CSSStyleDeclaration]: CSSStyleDeclaration[K] extends string ? K : never }[keyof CSSStyleDeclaration];
```

Comment:
Nice! You could also exclude `number`:

```suggestion
      = Exclude<{ [K in keyof CSSStyleDeclaration]: CSSStyleDeclaration[K] extends string ? K : never }[keyof CSSStyleDeclaration], number>;
```